PSIni/Private/Out-Key.ps1

```diff
@@ -43,17 +23,13 @@
     )
 
     begin {
-        $outFileParameter = @{
-            Append   = $Append
-            Encoding = $Encoding
-            FilePath = $FilePath
-            Force    = $Force
-        }
+        $outputLines = @()
```

Comment:
This function no longer supports processing input via pipeline.

let's remove the `begin`, `processing` and `end` blocks to be consistent with other private functions.